docs/features/metrics.rst

```diff
@@ -0,0 +1,24 @@
+.. _metrics:
+
+##############
+Metrics
+##############
+
+Metrics are values derived from one or more Data Assets that can be used to evaluate expectations or to summarize the
```

Comment:
Are we still using "Data Asset" and "Batch" interchangeably?

Reply:
I'm trying to migrate to batch == data, data asset == "validatable"...should this be changed?